src/main/java/com/BrewMES/demo/model/System.java

```diff
@@ -12,8 +13,17 @@ public static void main(String[] args) {
 		throw new UnsupportedOperationException();
 	}
 
+	public void setMachines(List<Machine> machines) {
+		this.machines = machines;
+	}
+
+	public Machine getCurrentMachine() {
+		return currentMachine;
+	}
+
+	// picks based on machine id
 	public void setCurrentMachine(int machine) {
```

Comment:
Perhaps the parameter should be renamed to int machineId? because isn't it the id that's put in and not the index?